.github/workflows/nightlies.yml

```
@@ -129,13 +129,7 @@ jobs:
                 jq --null-input --arg tag "$tag" '{ release: $tag }'
                 jq --null-input --argjson nightlies_revision "$nightlies_revision" '{ nightlies_revision: $nightlies_revision }'
                 jq --null-input --argjson build_revision "$buildRevision" '{ build_revision: $build_revision }'
-
-                if [[ "$tag" == version-2-0 ]]; then
-                  jq --null-input --argjson csources "nim-lang/csources_v2" '{ csources: $csources }'
-                fi
-                if [[ "$tag" == devel ]]; then
-                  jq --null-input --argjson csources "nim-lang/csources_v2" '{ csources: $csources }'
-                fi
+                jq --null-input --argjson csources "nim-lang/csources_v2" '{ csources: $csources }'
```

```suggestion
                jq --null-input --arg csources "nim-lang/csources_v2" '{ csources: $csources }'
```

Maybe `jq` made using `argjson` here into an error since the last time this triggered? This implies as such: https://stackoverflow.com/questions/77486468/jq-argjson-does-not-work-with-string-from-variable

---

This seems that has done the trick! Thanks!!